content/support-matrix/delfin/_index.md

```diff
@@ -22,5 +22,9 @@ Each driver support matrix includes below topics.
 6. Alert data matrix : Table illustrates fields available in the alert data
 7. Performance metric data: Table illustrates performance metrics supported for each resource.
 
+{{% notice note %}}
+Support matrix of drivers for hpe 3par, hitachi vsp, ibm storwize svc and unity, vnx, vplex models of dell emc are currently not updated here. 
```

Comment:
Please use appropriate capitalisation and case since these are brand names and acronyms.
for ex: hpe 3par -> HPE 3PAR

Reply:
yes let follow vendor naming convention in document  ( Dell EMC VNX, Dell EMC VPLEX, IBM Storewize/SVC, Hitachi VSP, HPE 3PAR)